lib/nile-agent/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
58],[240,45],[242,59],[243,45],[249,60],[250,1],[245,1],[248,61],[246,1],[251,1],[253,62],[254,63],[256,64],[257,1],[258,1],[259,1],[260,65],[261,66],[270,67],[269,68],[267,69],[268,70],[247,1],[271,1],[273,71],[274,72],[272,73],[275,74],[276,75],[277,76],[278,77],[279,78],[280,79],[281,80],[282,81],[283,82],[284,83],[285,64],[252,1],[286,1],[288,1],[289,84],[109,85],[110,85],[112,86],[113,87],[114,88],[115,89],[116,90],[117,91],[118,92],[119,93],[120,94],[121,95],[122,95],[123,96],[124,97],[125,98],[126,99],[111,100],[160,1],[127,101],[128,102],[129,103],[161,104],[130,105],[131,106],[132,107],[133,108],[134,109],[135,110],[136,111],[137,112],[138,113],[139,114],[140,114],[141,115],[142,116],[144,117],[143,118],[145,119],[146,120],[147,1],[148,121],[149,122],[150,123],[151,124],[152,125],[153,126],[154,127],[155,128],[156,129],[157,130],[158,131],[159,132],[290,1],[291,97],[292,1],[293,1],[294,1],[295,1],[296,1],[297,1],[302,133],[303,133],[304,133],[305,134],[306,133],[311,1],[308,135],[312,136],[310,133],[307,133],[309,135],[298,1],[301,137],[313,138],[314,1],[300,1],[315,1],[316,1],[317,1],[319,139],[318,1],[321,140],[255,1],[322,1],[323,63],[339,1],[337,141],[336,142],[327,143],[328,144],[335,145],[329,144],[330,143],[331,143],[332,143],[333,146],[326,147],[334,142],[325,1],[338,148],[340,1],[341,149],[324,1],[263,1],[299,1],[287,150],[266,151],[320,1],[46,1],[9,1],[10,1],[14,1],[13,1],[3,1],[15,1],[16,1],[17,1],[18,1],[19,1],[20,1],[21,1],[22,1],[4,1],[5,1],[26,1],[23,1],[24,1],[25,1],[27,1],[28,1],[29,1],[6,1],[30,1],[31,1],[32,1],[33,1],[7,1],[34,1],[35,1],[36,1],[37,1],[8,1],[38,1],[43,1],[44,1],[39,1],[40,1],[41,1],[42,1],[2,1],[1,1],[45,1],[12,1],[11,1]],"semanticDiagnosticsPerFile":[172,173,174,182,180,175,177,178,181,176,179,107,97,99,96,98,105,101,102,103,100,106,104,166,168,167,170,169,183,87,89,86,91,82,84,95,92,88,83,90,93,85,94,108,162,163,164,165,171,68,69,73,70,71,72,74,48,49,50,51,54,55,67,56,57,58,59,60,61,52,62,63,64,65,66,53,47,76,75,185,184,79,80,186,78,77,81,241,239,262,265,264,187,189,190,188,238,191,193,192,194,195,197,196,198,199,201,202,203,205,206,207,208,204,209,210,212,211,213,214,222,215,216,217,218,219,220,221,223,224,200,225,226,227,229,228,235,231,237,230,236,232,233,234,244,240,242,243,249,250,245,248,246,251,253,254,256,257,258,259,260,261,270,269,267,268,247,271,273,274,272,275,276,277,278,279,280,281,282,283,284,285,252,286,288,289,109,110,112,113,114,115,116,117,118,119,120,121,122,123,124,125,126,111,160,127,128,129,161,130,131,132,133,134,135,136,137,138,139,140,141,142,144,143,145,146,147,148,149,150,151,152,153,154,155,156,157,158,159,290,291,292,293,294,295,296,297,302,303,304,305,306,311,308,312,310,307,309,298,301,313,314,300,315,316,317,319,318,321,255,322,323,339,337,336,327,328,335,329,330,331,332,333,326,334,325,338,340,341,324,263,299,287,266,320,46,9,10,14,13,3,15,16,17,18,19,20,21,22,4,5,26,23,24,25,27,28,29,6,30,31,32,33,7,34,35,36,37,8,38,43,44,39,40,41,42,2,1,45,12,11]},"version":"4.7.4"}
```

Comment:
I think this should get added to the git ignore. I don't think it's useful to the package